index.html

```diff
@@ -104,7 +111,37 @@ <h2>-ui</h2>
           </ul>
         </div>
         <div class="main">
+          <div class="social-links">
+            <div class="nav-group">
+              <div class="nav-item">
+                <a target="_blank" href="https://twitter.com/kunal_tijare"
```

Comment:
Always add `rel="noopener"` or `rel="noreferrer"` to any links that open in new tab(`_blank`) for security and performance reasons.